Homework_2/templates/welcome.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Добро пожаловать</title>
+</head>
+<body>
+    <h2>Привет, {{ request.cookies['username'] }}!</h2>
+    <a href="/logout">Выйти</a>
```

Comment:
А тут нужно сделать так
```suggestion
    <form method="post">
        <button type = "submit">Выйти</button>
    </form>
```
либо так
```suggestion
    <a href="/">Выйти</a>
```
но тогда нужно убрать строки 12 и 25 в функции `welcome`